mozilla-mobile/focus-android/app/src/main/res/values/strings.xml

```diff
@@ -84,7 +84,11 @@
     <string name="share_dialog_title">Share via</string>
 
     <string name="notification_erase_title_android_14">Erase browsing history?</string>
-    <string name="notification_erase_text_android_14">Tap or clear this notification to securely erase your browsing history.</string>
+    <string name="notification_erase_text_android_14" moz:RemovedIn="124" tools:ignore="UnusedResources">Tap or clear this notification to securely erase your browsing history.</string>
+
+    <!-- Text shown in the notification description that pops up to remind the user that a private browsing session is active
+    for Android 14+. -->
+    <string name="notification_erase_text_android_14_1">Tap or swipe this notification to securely erase your browsing history.</string>
```

Comment:
The comment seems off when paired to this string. But could you help confirm @jonalmeida?

Reply:
@Delphine could you clarify what part seems off? This string is part of the notification because in Android 14+ you can dismiss "sticky" notifications now and this would end up killing the tab.

We used the same [string and comment from Fenix](https://searchfox.org/mozilla-mobile/rev/6fe38ba1c98e9d97bf74e9492df7acdbbbc11282/firefox-android/fenix/app/src/main/res/values/strings.xml#1260) that is shared here.

Reply:
Aha ok! I was unsure because the comment talks about private browsing notification ("to remind the user that a private browsing session is active"), but the string mentions erasing browsing history: "Tap or swipe this notification to securely erase your browsing history"

Reply:
Gotcha! In Focus, we erase all browsing history as all sessions are private, so maybe mentioning private browsing sessions in this context can be confusing?

Maybe if we take out "private":
> Text shown in the notification description that pops up to remind the user that a browsing session is active for Android 14+.

Reply:
Yes, I think that would help, thank you! 😄